Taskfile.yaml

```diff
@@ -196,6 +196,38 @@ tasks:
         echo "✅ Redeployment complete!"
         echo "Check logs with: task test-infra:kubectl -- logs -n milo-system -l app.kubernetes.io/name=milo-controller-manager"
 
+  dev:install-observability:
+    desc: "Install test-infra observability, then apply Milo-specific scraping manifests"
+    silent: true
+    cmds:
+      - |
+        set -euo pipefail
+        echo "🔧 Installing base observability stack via test-infra…"
+        task test-infra:install-observability
+
+        echo "⏳ Waiting for Prometheus CRDs to be Established…"
+        for crd in \
+          servicemonitors.monitoring.coreos.com \
+          podmonitors.monitoring.coreos.com \
+          probes.monitoring.coreos.com \
+          prometheuses.monitoring.coreos.com \
+          prometheusrules.monitoring.coreos.com \
+          alertmanagers.monitoring.coreos.com \
+          thanosrulers.monitoring.coreos.com
+        do
+          echo "  • $crd"
+          task test-infra:kubectl -- wait --for=condition=Established "crd/${crd}" --timeout=120s
+        done
```

Comment:
Is this needed here? I would expect this to be done as part of `task test-infra:install-observability`

Reply:
good catch